polyfills.js

```diff
@@ -1,4 +1,4 @@
-var constants = require('constants')
+var constants = require('fs').constants
```

Comment:
This module still supports ancient versions of node.js.  I suspect we need:
```suggestion
var constants = require('fs').constants || require('constants')
```

Reply:
Thank you for the advice.

> still supports ancient versions of node.js.

I'm curious what version is the oldest supported version.

Reply:
> Thank you for the advice.
> 
> > still supports ancient versions of node.js.
> 
> I'm curious what version is the oldest supported version.

All of them.  The next major will bump the requirements but no ETA for that.